Services/Interfaces/web_interface/flask_util/file_services.py

```diff
@@ -20,7 +20,7 @@
 
 def send_and_remove_file(file_path, attachment_filename):
     try:
-        return flask.send_file(file_path, as_attachment=True, attachment_filename=attachment_filename, cache_timeout=0)
+        return flask.send_file(file_path, as_attachment=True, download_name=attachment_filename, max_age=0)
```

Comment:
thanks for the fix, however could you rebase of dev please ? This is fixed in the dev branch :)